src/services/UserService.js

```diff
@@ -22,14 +23,21 @@ const createUser = async (phoneNumber, installationId) => {
 
   const newUser = new Parse.User();
   newUser.setUsername(uuidv4());
-  newUser.setPassword(generatePassword(installationId));
+  const hashcode = hat();
+  newUser.setPassword(generatePassword(hashcode));
+
+  const userAttributes = {
+    phoneNumber,
+    language: 'en',
+    hashcode,
+  };
+  if (!installationId) {
```

Comment:
Is this still needed?

Reply:
Yes! If you have an installation id It will create a session for this installation.

Reply:
Got it.